Britannia/events/BTWK_storyline_events.txt

```diff
@@ -0,0 +1,390 @@
+
+### ARTHUR REGENCY CHAIN ###
+# BTWK_story_event.0008 = Regent decides to give power to Arthur
+# BTWK_story_event.0009 = Arthur accepts regency
+# BTWK_story_event.0010 = Notify Mordred
+# BTWK_story_event.0011 = Notify other vassals
+# BTWK_story_event.0012 = Set Flag
+
+
+############## UTHER'S DEATH and ARTHUR'S RETURN ##############
+
+
+BTWK_story_event.0001 = {  # Triggered by on_game_start on_action incase Uther's pneumonia is cured. Possibly make it non-hidden also. "Second bout of illness..."
+	title = BTWK_story_event.0001.t
+	desc = BTWK_story_event.0001.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+	right_portrait = {
+		character = character:1  # Uther
+		animation = worry
+	}	
+	lower_left_portrait = {
+		character = character:5  # Mordred ap Mordred
+		animation = no
+	}
+	lower_right_portrait = {
+		character = character:2  # Merlin
+		animation = no
+	}
+	lower_center_portrait = {
+		character = character:20  # Tewdric
+		animation = no
+	}
+
+	
+	option = {
+		name = BTWK_story_event.0001.a
+		trigger = {
+			NOT = { has_character_flag = utherillness }
+			is_alive = yes	
+		}
+		character:1 = {
+			add_trait = pneumonic
+			add_character_flag = utherillness
+			trigger_event = {
+				id = BTWK_story_event.0002
+				days = {100 180}
+			}
+		}
+		
+	}
+}
+
+BTWK_story_event.0002 = {  # Should probably change this to make it non-hidden & a unique end of life event for Uther.
+	type = character_event
+	title = BTWK_story_event.0002.t
+	desc = BTWK_story_event.0002.desc
+	# More fitting music cue/sounds needed
+	theme = skull
+
+
+	right_portrait = {
+		character = character:1
+		animation = worry
+	}
+	
+	option = {
+		name = BTWK_story_event.0002.a
+		trigger = {
+			this = character:1	
+			has_character_flag = utherillness
+			is_alive = yes
+			NOT = { has_character_flag = utherincapable }
+		}
+		character:1 = {
+			add_trait = incapable
+			add_character_flag = utherincapable
+		}
+		
+	}
+}
+
+BTWK_story_event.0003 = {  #Sub-event that delays Arthur's return, & fires "Uther is dead" event.
+
+	type = character_event
+	hidden = yes
+	
+	trigger = {
+		character:1 = { is_alive = no }
+		has_global_variable = uther_is_dead
+	}
+	immediate = {
+		remove_global_variable = uther_is_dead
+		#  Fires Arthur's return HIDDEN EVENT  #
+		trigger_event = {
+			id = BTWK_story_event_0004
+		}
+	}
+}
+
+BTWK_story_event.0004 = {
+
+	type = character_event
+	hidden = yes
+
+	immediate = {
+
+		#  Arthur gets d_durnac & vassals  #
+		create_title_and_vassal_change = {
+			type = granted
+			save_scope_as = arthur_pendragon_durnac
+			add_claim_on_loss = yes
+		}
+		title:d_centdumnonia = {
+			every_de_jure_county = {
+				limit = {
+					holder = character:5
+				}
+				change_title_holder = {
+					holder = character:3
+					change = scope:arthur_pendragon_durnac
+				}
+			}
+			change_title_holder = {
+				holder = character:3
+				change = scope:arthur_pendragon_durnac
+			}
+			every_de_jure_county_holder = {
```

Comment:
As a sanity check, we probably also want to check that these vassals are a) actually count tier and b) their primary title is in d_centdumnonia (as they shouldn't be vassalized under Arthur if either of these are false)